packages/modules/fsm/src/pages/employee/DsoDashboard.js

```diff
@@ -20,13 +21,25 @@ const DsoDashboard = () => {
   const tenantId = Digit.ULBService.getCurrentTenantId();
   const { t } = useTranslation();
 
-  const [info, setInfo] = useState({
-    [t("ES_PENDING")]: "-",
-    [t("ES_NEARING_SLA")]: "-",
-  });
+  const [info, setInfo] = useState({});
+  const [total, setTotal] = useState("-");
+  const [loader, setLoader] = useState(true);
```

Comment:
we can use if condition to render loader

Reply:
that would be inconvinient